fantasi/Tuura/Fantasi/Main.hs

```diff
@@ -14,9 +14,9 @@ main = do
         simulationEnvVhdlPath = optSimName options
 
     -- parse graph
-    let pangraph = P.unsafeParse (pack graphMLPath)
-    let graphVHDL   = (unpack . VHDL.writeGraph) pangraph
-    let simEnvVHDL  = (unpack . VHDL.writeEnvironment) pangraph
+    pangraph <- P.unsafeParse <$> readFile graphMLPath
```

Comment:
Can we switch to using `parse` here, catching and reporting an error if the parsing goes wrong?